tailwind.config.js

```diff
@@ -28,17 +28,42 @@ module.exports = {
           "base-200": "#f1f5f9",
           "base-300": "#e2e8f0",
 
-          primary: "#4885ed",
-          "primary-content": "#E8F0FE",
-          secondary: "#3cba54",
-          accent: "#f4c20d",
-          neutral: "#0f172a",
-          "neutral-content":"#AEB0B3",
+          primary: "#4285F4",
+          ".primary-fade": {
+            "background-color": "#8AB4F8",
+          },
+          ".primary-focus": {
+            "background-color": "#1A73E8",
+          },
```

Comment:
focus states should **not** be in this format. They are supported as CSS variables by DaisyUI, so you should use the format 
```js
"primary-focus": "#1A73E8"
```